tasks/linter.py

```diff
@@ -406,7 +406,7 @@ def releasenote(ctx):
             if not github.contains_release_note(pr_id):
                 print(
                     f"{color_message('Error', 'red')}: No releasenote was found for this PR. Please add one using 'reno'"
-                    ", see https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno"
+                    ", see https://github.com/DataDog/datadog-agent/blob/main/docs/public/guidelines/contributing.md#reno"
```

Comment:
```suggestion
                    ", see https://datadoghq.dev/datadog-agent/guidelines/contributing/#reno"
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ We're trying to use https://datadoghq.dev/datadog-agent/ now (which is a properly displayed version of the `https://github.com/DataDog/datadog-agent/blob/main/docs/public` folder !